<launch>

    <arg name="file" default="$(find eigenbot_Sep2020)/urdf/autoXACRO.urdf" />
    <arg name="model_name" default="autoXACRO" />
    <node name="spawn_robot" pkg="gazebo_ros" type="spawn_model" 
        args="-file $(arg file) -urdf -model $(arg model_name)" />

</launch>
